README.md

```
@@ -1,96 +1,150 @@
+Shuffle is visible for psychological damage
+UI is minimalist and slightly hostile
+
+Files:
+index.html – Structure
+style.css – Visual pain
+script.js – Game logic and timer
+
 # Installation
-[commands]
+
+git clone https://github.com/your-username/too-many-buttons.git
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)